Empire/Empire.cpp

```diff
@@ -318,22 +318,6 @@ void Empire::AdoptPolicy(const std::string& name, const std::string& category,
 }
 
 void Empire::UpdatePolicies(bool update_cumulative_adoption_time) {
-    // remove any unrecognized policies and uncategorized policies
-    auto policies_temp = m_adopted_policies;
-    for (auto& [policy_name, adoption_info] : policies_temp) {
-        const auto* policy = GetPolicy(policy_name);
-        if (!policy) {
-            ErrorLogger() << "UpdatePolicies couldn't find policy with name: " << policy_name;
-            m_adopted_policies.erase(policy_name);
-            continue;
-        }
-
-        if (adoption_info.category.empty()) {
-            ErrorLogger() << "UpdatePolicies found policy " << policy_name << " in empty category?";
-            m_adopted_policies.erase(policy_name);
-        }
-    }
-
```

Comment:
As far as I understand it, doing these checks after loading, i.e. after a possible change in rules, should be enough. Not really sure about the sure about the second one, though.